docs/rfc/rfc-1.adoc

```diff
@@ -234,6 +234,22 @@ and blinding factor, and then request that TBTC gets minted ASAP. Checking the
 dApp, they can see that they should expect TBTC in their provided wallet
 address in 3 hours with no further interaction.
 
+==== Fraud Proof
+
+There are only two valid types of transactions for a sweep:
+
+1) A collection of deposit UTXO's plus the main UTXO for the wallet are the
+inputs are unlocked using the `signingGroupPubkey` and locked under a single
+UTXO using the same `signingGroupPubkey`. This is a normal non-fraudulent sweep.
```

Comment:
Diving a bit deeper, is it fine to say: as long as UTXO uses `signingGroupPubKey`, this is not a fraud? I think this could simplify the implementation because we don't need to track UTXO, just extract `signingGroupPubKey` from the fraud proof and compare.

Reply:
It's slightly more complicated than that. If we go `signingGroupPubKey` => `signingGroupPubKey` then it's fine. If we go `refundPubkey` => `anything` then it's fine. If we go `signingGroupPubKey` => `anything else` then it's fraud.

The explanation on line 249 is the summary, and the stuff from line 241-247 is the technical explanation for why that summary is true!